packages/@okta/vuepress-site/docs/guides/authenticators-okta-verify/main/java/commonsteps.md

```diff
@@ -0,0 +1,105 @@
+
+After a user initiates the sign-in flow by submitting their username and password add the following code:
+
+```java
+// create an IDX Authentication Wrapper (can be an application scoped singleton)
+IDXAuthenticationWrapper idxAuthenticationWrapper = new IDXAuthenticationWrapper();
+// begin transaction
+// The proceedContext needs to be persisted between interactions (for example, stored in a HttpSession for web apps)
+ProceedContext proceedContext = idxAuthenticationWrapper.begin().getProceedContext();  // TODO this needs request context
+// set the user's credentials
+AuthenticationOptions authenticationOptions = new AuthenticationOptions(username, password);
+// start authentication flow
+AuthenticationResponse authenticationResponse =
+    idxAuthenticationWrapper.authenticate(authenticationOptions, proceedContext);
+```
+
+#### 2: Handle the response from the sign-in flow
+
+The `AuthenticationResponse` contains the current state of the authentication process. It could be a `SUCCESS` or may indicate that more information is needed to complete the flow, for example, require an additional factor.
+
+Query the `authenticationStatus` property of the `AuthenticationResponse` object returned by `authenticate()` method to discover the current status of the authentication process.
+
+```java
+// Update the context each time a request is made to Okta
+// make sure it is persisted for web applications
+proceedContext = authenticationResponse.getProceedContext();
+// get the authenticationStatus
+AuthenticationStatus authenticationStatus = authenticationResponse.getAuthenticationStatus();
+switch (authenticationStatus) {
+    case SUCCESS:
+        … your code …
+    case PASSWORD_EXPIRED:
+        … your code …
+}
+```
+
+#### 3: Display a list of possible authenticator factors
+
+Your application will need to display the list of authenticators (including Okta Verify).
+
+```java
+private AuthenticationResponse selectAuthenticator(AuthenticationResponse authenticationResponse) {
+
+    List<Authenticator> authenticators = authenticationResponse.getAuthenticators();
+
+    // show authenticators to user
+    IntStream.range(0, authenticators.size()).forEach(index -> {
+        console.writer().println(index + " - " + authenticators.get(index).getLabel());
+    });
+
+    // prompt user to select a factor
+    String selection = console.readLine("Select a Factor: ");
+    Authenticator authenticator = authenticators.get(Integer.parseInt(selection));
+
+    return idxAuthenticationWrapper.selectAuthenticator(proceedContext,  authenticator);
+}
+```
+
+The available authenticators are listed along with Okta Verify.
+
+At this point, the next steps differ depending on whether you are working with an enrollment flow or a challenge flow. Go to the specific flows (enroll with [QR Code](#integrate-enrollment-using-qr-code) or enroll with [other channels](#integrate-enrollment-using-other-channels)) to understand how.
+
+### Polling Okta
```

Comment:
The sequence diagram is missing for this section.

Reply:
The original diagram doesn't fit in this guide. As it's just a single polling method in this example